frontend/package.json

```diff
@@ -47,8 +47,9 @@
     "eslint-plugin-sonarjs": "^0.5.0",
     "gatsby-plugin-asset-path": "^3.0.2",
     "gatsby-plugin-root-import": "^2.0.5",
-    "lint-staged": "^10.2.11",
-    "prettier": "^1.19.1",
+    "husky": "^4.3.0",
```

Comment:
Add `lint-staged` and `husky` for automatically running prettier when committing changes

Also upgrading `prettier` to the latest version